manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/api/ManagerInternalPlatformExceptionMapper.java

```diff
@@ -0,0 +1,36 @@
+package com.redhat.service.smartevents.manager.v2.api;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.ext.Provider;
+
+import com.redhat.service.smartevents.infra.core.exceptions.BridgeErrorService;
+import com.redhat.service.smartevents.infra.core.exceptions.definitions.platform.InternalPlatformException;
+import com.redhat.service.smartevents.infra.core.exceptions.mappers.InternalPlatformExceptionMapper;
+import com.redhat.service.smartevents.infra.core.models.responses.ErrorResponse;
+import com.redhat.service.smartevents.infra.v2.api.V2APIConstants;
+
+@Provider
+@ApplicationScoped
```

Comment:
same here

Reply:
Same here too.

Reply:
Same here too.

Reply:
Removed until Error Handling refactor is complete.